.travis.yml

```diff
@@ -39,38 +32,36 @@ before_install:
   - git config --global user.name "Travis Deploy"
   - git config --global user.email "rmg_dev@mit.edu"
 
-install:
-  # Clone RMG-database
-  - git clone https://github.com/ReactionMechanismGenerator/RMG-database.git
-  - cd RMG-Py
-  # Create and activate environment
-  - conda env create -q -f $ENV_FILE
-  - source activate rmg_env
-  # Install codecov for coverage report
-  - conda install -y -c conda-forge codecov
-  - conda list
-  # Setup MOPAC license key
-  - yes 'Yes' | $HOME/miniconda/envs/rmg_env/bin/mopac $MOPACKEY > /dev/null
-  # Install Q2DTor
-  - git clone -b arkane https://github.com/mjohnson541/Q2DTor.git external/Q2DTor
-  # Compile RMG
-  - make
-
-script:
-  - make test-unittests
-  - make test-functional
-  - make test-database
-
-after_success:
-  - codecov
-  - bash ./deploy.sh
-
 jobs:
   include:
+    - stage: test
+      install:
+        # Clone RMG-database
+        - git clone https://github.com/ReactionMechanismGenerator/RMG-database.git
+        - cd RMG-Py
+        # Create and activate environment
+        - conda env create -q -f environment_py3.yml
+        - source activate rmg_env
+        # Install codecov for coverage report
+        - conda install -y -c conda-forge codecov
+        - conda list
+        # Setup MOPAC license key
+        - yes 'Yes' | $HOME/miniconda/envs/rmg_env/bin/mopac $MOPACKEY > /dev/null
+        # Install Q2DTor
+        - git clone -b arkane https://github.com/mjohnson541/Q2DTor.git external/Q2DTor
+        # Compile RMG
+        - make
+      script:
+        - make test-unittests
+        - make test-functional
+        - make test-database
+      after_success:
+        - codecov
+        - bash ./deploy.sh
     - stage: documentation
       install:
         - cd RMG-Py
-        - conda env create -q -f environment_linux.yml
+        - conda env create -q -f environment_py3.yml
```

Comment:
It could be part of this PR. I started working on it, but then realized that tests and documentation would need to be updated as well.

My plan is to remove the old environment files completely and rename `environment_py3.yml` to `environment.yml`.